protodroid/src/main/java/id/lukasdylan/grpc/protodroid/internal/ui/ProtodroidActivity.kt

```diff
@@ -92,4 +93,13 @@ internal class ProtodroidActivity : ComponentActivity() {
             }
         }
     }
+
+    // https://issuetracker.google.com/issues/139738913
+    override fun onBackPressed() {
+        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.Q && isTaskRoot) {
```

Comment:
if you override this function, have you test it when you are in DetailScreen??

Reply:
Changed approach here, works with both DetailScreen and MainScreen